server/src/domain/facial-recognition/facial-recognition.services.ts

```diff
@@ -68,7 +68,14 @@ export class FacialRecognitionService {
       return false;
     }
 
-    const faces = await this.machineLearning.detectFaces(machineLearning.url, { imagePath: asset.resizePath });
+    const {
+      machineLearning: { facialRecognition },
+    } = await this.configCore.getConfig();
```

Comment:
You already have the full config from line 61, so I think you don't need to get it again?

Reply:
Good catch!